src/EnergyPlus/DXCoils.cc

```diff
@@ -9922,8 +9921,11 @@ Label50: ;
 		Real64 Slope( 0.0 ); // Calculated Slope used while hunting for Tadp
 		Real64 Tolerance; // Convergence tolerance for CBF calculations
 		Real64 HTinHumRatOut; // Air enthalpy at inlet air temp and outlet air humidity ratio [J/kg]
+		Real64 AirMassFlowRate; // the standard air mass flow rate at the given capacity [kg/s]
+		Real64 adjustedSHR; // SHR calculated using adjusted outlet air properties []
 		static bool CBFErrors( false ); // Set to true if errors in CBF calculation, fatal at end of routine
 
+		AirMassFlowRate = AirVolFlowRate * PsyRhoAirFnPbTdbW( StdPressureSeaLevel, InletAirTemp, InletAirHumRat, RoutineName );
```

Comment:
Fundamental change is right here. Pass in volume and convert to standard mass flow. Function used to pass in mass flow rate, sometimes at STP and other time at elevation. Now it's consistent. Changing back to how it's done now simply means changing StdPressureSeaLevel to StdBaroPress, not that we'd ever do that.